packages/components/src/internal/util/Date.spec.ts

```diff
@@ -74,3 +74,15 @@ describe('formatDateTime', () => {
         expect(formatDateTime(testDate, 'America/New_York', 'DDYYYYMM')).toBe('06202008');
     });
 });
+
+describe('getJsonDateTimeFormatString', () => {
+    test('without date', () => {
+        expect(getJsonDateTimeFormatString(undefined)).toBe(null);
+        expect(getJsonDateTimeFormatString(null)).toBe(null);
+    });
+
+    test('with date', () => {
+        expect(getJsonDateTimeFormatString(new Date('2021-12-03 00:00'))).toBe('2021-12-03 00:00:00');
```

Comment:
Is this the format produced by react-datepicker?

Reply:
are you talking about the `new Date('2021-12-03 00:00')` value? No, that is just a way for me to create a JS date object which is a constant value that I can have the test check the result for. The react-datepicker just returns a JS date object, so that part shouldn't matter.